storage/ringbuffer/adapter_test.go

```diff
@@ -18,7 +18,7 @@ func TestReadFromNonExistingApp(t *testing.T) {
 	if messages != nil {
 		t.Error("Expected no messages, but got some")
 	}
-	if err == nil || err.Error() != fmt.Sprintf("Could not find logs for '%s'", app) {
+	if err == nil || err.Error() != fmt.Sprintf("Could not find logs for '%s'. No ringbuffer existed for '%s'.", app, app) {
```

Comment:
after https://github.com/deis/logger/pull/96/files#r68310164 is done, then this could check the error type:

```go
_, errOK := err.(ErrNoRingBufferExistedForApp)
if err == nil || !errOK {
  //...
}